thing-url-adapter.js

```diff
@@ -516,6 +528,32 @@ class ThingURLAdapter extends Adapter {
     this.knownUrls = {};
     this.savedDevices = new Set();
     this.pollInterval = POLL_INTERVAL;
+    this.authData = {}; // set in loadthingurladapter
+  }
+
+  getHeaders(url, contentType = false) {
+    const headers = {Accept: 'application/json'};
+    if (contentType) {
+      headers['Content-Type'] = 'application/json';
+    }
+    for (const i in this.authData) {
```

Comment:
This would be a little cleaner, and would remove the need for the proceeding `hasOwnProperty` check:

`for (const [url, authData] of Object.entries(this.adapter.authData))`

Reply:
this too